### PR TITLE
Feature/cases contact tab export

### PR DESCRIPTION
**Pull request by constantincrismaruvitagroup**

### CODE REVIEW

File: src/app/cases/case_components/case-contacts/case-contacts.component.scss [hunk 1]
```diff
@@ -6,3 +6,10 @@
   font-size: 34px;
   font-weight: 500;
 }
+
+.actions-section{
+  display: flex;
+  justify-content: end;
+  padding: 32px;
+  background-color: #fff;
```
Comment (sandorbanvitagroup) [inline src/app/cases/case_components/case-contacts/case-contacts.component.scss]: Please use a variable for the color.

File: src/app/cases/case_components/case-contacts/case-contacts.component.ts [hunk 2]
```diff
@@ -36,4 +73,50 @@ export class CaseContactsComponent {
...
+  }
+
+  exportBasicCase(): void {
+    this.notificationService.prompt({
+      title: this.translateService.instant('captions.exportBasic'),
+      message: this.translateService.instant('strings.infoDownloadExport'),
+      maxWidth: SMALL_NOTIFICATION_MODAL_WIDTH,
+    });
+
+    this.exportService.executeExport(EXPORT_TYPES.BASIC, API_ROUTE_CASES.EXPORT);
```
Comment (danielrus) [inline src/app/cases/case_components/case-contacts/case-contacts.component.ts]: We're no the cases contact tab, so I expect this to be an export of contacts. This should be API_ROUTE_CONTACTS.

File: src/app/cases/case_components/case-contacts/case-contacts.component.ts [hunk 3]
```diff
@@ -36,4 +73,50 @@ export class CaseContactsComponent {
...
+  }
+
+  exportDetailedCase(): void {
+    this.notificationService.prompt({
+      title: this.translateService.instant('captions.exportDetailed'),
+      message: this.translateService.instant('strings.infoDownloadExport'),
+      maxWidth: SMALL_NOTIFICATION_MODAL_WIDTH,
+    });
+
+    this.exportService.executeExport(EXPORT_TYPES.DETAILED, API_ROUTE_CASES.EXPORT);
```
Comment (danielrus) [inline src/app/cases/case_components/case-contacts/case-contacts.component.ts]: same as above

File: angular.json [hunk 4]
```diff
@@ -140,6 +140,7 @@
   },
   "defaultProject": "sormas",
   "cli": {
+    "analytics": false,
```
Comment (adriandumavitagroup) [inline angular.json]: is this necessary?

File: angular.json [hunk 5]
```diff
@@ -140,6 +140,7 @@
...
     "defaultCollection": "@angular-eslint/schematics",
     "cache": {
       "enabled": false
```
Comment (adriandumavitagroup) [inline angular.json]: same here ; why change this?

File: src/app/cases/case_components/case-contacts/case-contacts.component.ts [hunk 6]
```diff
@@ -20,10 +34,33 @@ export class CaseContactsComponent {
   configKey = CONFIG_CASES;
   preSetFilters: Filter[];
   formIdFilters = CONTACT_FILTERS_FORM_ID;
+  actionsMore: NavItem[] = [
+    {
+      role: NavItemRole.ACTION,
+      name: 'captions.exportBasic',
+      action: ACTIONS_CONTACT.BASIC_EXPORT,
```
Comment (adriandumavitagroup) [inline src/app/cases/case_components/case-contacts/case-contacts.component.ts:41]: you should have here ACTION_CASE

Comment (adriandumavitagroup) [reply src/app/cases/case_components/case-contacts/case-contacts.component.ts:41]: and for the other bellow

File: src/app/cases/case_components/case-contacts/case-contacts.component.ts [hunk 7]
```diff
@@ -36,4 +73,50 @@ export class CaseContactsComponent {
...
+  }
+
+  exportBasicCase(): void {
+    this.notificationService.prompt({
+      title: this.translateService.instant('captions.exportBasic'),
+      message: this.translateService.instant('strings.infoDownloadExport'),
+      maxWidth: SMALL_NOTIFICATION_MODAL_WIDTH,
+    });
+
+    this.exportService.executeExport(EXPORT_TYPES.BASIC, API_ROUTE_CONTACTS.EXPORT);
```
Comment (adriandumavitagroup) [inline src/app/cases/case_components/case-contacts/case-contacts.component.ts:94]: you should have API_ROUTE_CASES here

File: src/app/cases/case_components/case-contacts/case-contacts.component.ts [hunk 8]
```diff
@@ -36,4 +73,50 @@ export class CaseContactsComponent {
...
+  }
+
+  exportDetailedCase(): void {
+    this.notificationService.prompt({
+      title: this.translateService.instant('captions.exportDetailed'),
+      message: this.translateService.instant('strings.infoDownloadExport'),
+      maxWidth: SMALL_NOTIFICATION_MODAL_WIDTH,
+    });
+
+    this.exportService.executeExport(EXPORT_TYPES.DETAILED, API_ROUTE_CONTACTS.EXPORT);
```
Comment (adriandumavitagroup) [inline src/app/cases/case_components/case-contacts/case-contacts.component.ts:104]: you should have API_ROUTE_CASES here

File: src/app/cases/case_components/case-contacts/case-contacts.component.ts [hunk 9]
```diff
@@ -36,4 +73,50 @@ export class CaseContactsComponent {
...
+  }
+
+  onActionSelected(event: any): void {
+    switch (event) {
+      case ACTIONS_CONTACT.BASIC_EXPORT:
```
Comment (adriandumavitagroup) [inline src/app/cases/case_components/case-contacts/case-contacts.component.ts:109]: you should have ACTIONS_CASE here and also bellow

**Review by adriandumavitagroup — APPROVED** [review]

**Review by danielrus — APPROVED** [review]

**Pull request merged**